### PR TITLE
tiltfile: add os.path.relpath

### PR DESCRIPTION
**Pull request by maiamcc**
this would have been useful for the [Tests In Tilt code snippets](https://docs.tilt.dev/tests_in_tilt.html#go-tests-by-package) i was writing for docs, so I implemented it

### CODE REVIEW

**Review by nicks — APPROVED** [review]

File: internal/tiltfile/os/os.go [hunk 1]
```diff
@@ -164,6 +168,17 @@ func abspath(t *starlark.Thread, path string) (starlark.Value, error) {
 	return starlark.String(starkit.AbsPath(t, path)), nil
 }
 
+func relpath(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var basepath, targpath string
+	if err := starkit.UnpackArgs(thread, fn.Name(), args, kwargs,
+		"basepath", &basepath,
```
Comment (landism) [inline internal/tiltfile/os/os.go:174]: might wanna swap the arg order to allow compatibility with python

https://docs.python.org/3/library/os.path.html#os.path.relpath

**Review by landism — APPROVED** [review]

File: internal/tiltfile/os/os.go (hunk 1 above)
Comment (landism) [reply internal/tiltfile/os/os.go:174]: and it looks like defaulting the basepath to cwd would even be good for the docs!

Comment (maiamcc) [reply internal/tiltfile/os/os.go:174]: oh ugh you're right -- i did it this way to mirror the Go func but this makes much more sense 👍

**Comment by victorwuky** [comment]
@maiamcc : Should we add an entry to https://docs.tilt.dev/api.html for this change?

**Comment by maiamcc** [comment]
Yep, was waiting for the release to go out, just merged https://github.com/tilt-dev/tilt.build/pull/824

**Comment by victorwuky** [comment]
Thanks @maiamcc

**Pull request merged**